### PR TITLE
[android] fix for hidden update dialog buttons

### PR DESCRIPTION
**Pull request by marcelinkaaa**
The buttons of the Update Dialog we invisible when more bullet points were used - pushing them down. Switched the layout to relativelayout.

### CODE REVIEW

File: android/res/layout/dialog_default_update.xml [hunk 1]
```diff
@@ -17,56 +17,63 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 -->
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (aldenml) [inline android/res/layout/dialog_default_update.xml]: to @gubatron :)

File: android/res/layout/dialog_default_update.xml [hunk 2]
```diff
@@ -17,56 +17,63 @@
...
+        style="@style/DefaultDialogTheme.MainText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_alignParentLeft="true"
+        android:layout_below="@+id/dialog_default_update_title"
+        android:paddingBottom="10dp"
+        android:paddingLeft="15dp"
+        android:paddingRight="15dp"
+        android:paddingTop="10dp" />
+</RelativeLayout>
```
Comment (aldenml) [inline android/res/layout/dialog_default_update.xml]: Remember change the formatting (add a newline to the end of files)

File: android/res/layout/dialog_default_update.xml (hunk 1 above)
Comment (gubatron) [inline android/res/layout/dialog_default_update.xml]: my favorite. Way to go @marcelinkaaa, I was about to do this.

File: android/res/layout/dialog_default_update.xml (hunk 2 above)
Comment (gubatron) [inline android/res/layout/dialog_default_update.xml]: I'll never understand why these relative layouts have to add everything backwards

**Pull request merged**